superset-frontend/src/explore/components/controls/DateFilterControl/frame/AdvancedFrame.tsx

```diff
@@ -20,6 +20,7 @@ import React from 'react';
 import { t } from '@superset-ui/core';
 import { SEPARATOR } from 'src/explore/dateFilterUtils';
 import { Input } from 'src/common/components';
+import { InfoTooltipWithTrigger } from '@superset-ui/chart-controls';
```

Comment:
The reason I added InfoTooltipWithTrigger instead of InfoTooltip is from style
Start = InfoTooltip
End = InfoTooltipWithTrigger
![image](https://user-images.githubusercontent.com/8277264/106437100-921d1080-647d-11eb-9ff1-32e03c1318ba.png)

and it doesn't make sense to create custom styles for control label.